pkg/kubelet/containerdshim/containerd_container.go

```diff
@@ -151,26 +173,39 @@ func (cs *containerdService) StartContainer(containerID string) error {
 	if containerID == "" {
 		return fmt.Errorf("containerID should not be empty")
 	}
-	if _, err := cs.cdClient.Start(gocontext.Background(), &execution.StartRequest{
-		ID: containerID,
-	}); err != nil {
-		glog.V(2).Infof("StartContainer for %s failed with %v", containerID, err)
-		return err
-	}
-	return nil
+	_, err := cs.cdClient.Start(gocontext.Background(), &execution.StartRequest{ID: containerID})
+	return err
 }
 
 // StopContainer stops a running container with a grace period (i.e., timeout).
 // P0
 func (cs *containerdService) StopContainer(containerID string, timeout int64) error {
-	return fmt.Errorf("not implemented")
+	glog.V(2).Infof("StopContainer called with %s", containerdID)
+	if containerID == "" {
+		return fmt.Errorf("containerID should not be empty")
+	}
+	c, err := getShimClient(containerID)
```

Comment:
Ok, so they have not yet implemented Stop, Signal, Pause, or Resume... The only ways we could do this would be to fix containerd, implement our own signal routine to kill the init proc of the container, or shell out to runc to have it kill the container for us.